Lib/test/test_complex.py

```diff
@@ -0,0 +1,712 @@
+import unittest
+import Lib.test.librptest as rpt
```

Comment:
```suggestion
import test.librptest as rpt
```
We won't always be running from the root of the repo, which would put `Lib` under the pythonpath as a submodule, but `Lib` should already be in `RUSTPYTHONPATH`, so you don't need it here.

Reply:
obviously...